placement/staged_placement_watcher.go

```diff
@@ -28,6 +28,7 @@ import (
 	"github.com/m3db/m3cluster/kv"
 	"github.com/m3db/m3cluster/kv/util/runtime"
 	"github.com/m3db/m3x/clock"
+	"github.com/m3db/m3x/log"
```

Comment:
FYI please remember to update the matchers in `m3metrics/matcher` that are using `kv.Value` as well. They are currently used in collector/query/indexer.